lib/util/git/index.ts

```diff
@@ -628,18 +628,24 @@ export async function isBranchModified(branchName: string): Promise<boolean> {
   logger.debug('branch.isModified(): using git to calculate');
 
   await syncGit();
-  // Retrieve the author of the most recent commit
-  let lastAuthor: string | undefined;
+  // Retrieve the commit authors
+  let lastAuthors: string[];
   try {
-    lastAuthor = (
-      await git.raw([
-        'log',
-        '-1',
-        '--pretty=format:%ae',
-        `origin/${branchName}`,
-        '--',
-      ])
-    ).trim();
+    const defaultBranch = await getDefaultBranch(git);
```

Comment:
you need to compare to base branch instead because renovate can work on multiple branches